db/seeds.rb

```diff
@@ -16,5 +16,5 @@
 if Rails.env.development?
   user = User.where(email: 'admin@nimblehq.co').first_or_create(Fabricate.attributes_for(:admin, email: 'admin@nimblehq.co'))
 
-  Fabricate.times(100, :keyword, user: user)
+  Fabricate.times(100, :keyword_with_result, user: user)
```

Comment:
I think we dont have a factory for `:keyword_with_result `

Reply:
Well catched! It's a commit made in the wrong branch =/ 
Thanks!!

Reply:
Fixed!